### DISCUSSION TITLE
Seeing output in unraid docker?

### DISCUSSION THREAD

**Discussion by jruben4** [merbanan/rtl_433#3017] [Q&A]
I'm trying to get a nice output of received data (like the example on the top of the .readme - I've tried various -v -vv -vvv flags.
I'm running this in an unraid docker.  Should I see the output in the log file for the container, or on the console somehow?
Thanks!

**Comment by zuckschwerdt** [comment]
Just starting without any arguments should give that output. Make sure you don't have a rtl_433.conf with override like `output json`.
Otherwise force that output with `-F kv`.
Also use `-Y autolevel -M level -M noise -M time:usec -v` for best output.

**Reply by jruben4** [reply]
And I should expect to see it in the docker container log?

**Comment by zuckschwerdt** [comment]
If no color-capable tty (or `NO_COLOR` is set) with a given width is detected there will be no color and each key/value will be on a new line. You can force color with `RTL433_COLOR=always`.
